src/mavsdk/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -458,9 +458,9 @@ class Telemetry : public PluginBase {
      */
     struct VehicleStatus {
         bool manual_control_signal_loss{false}; /**< @brief True if manual control signal is loss */
-        bool data_link_loss{false}; /**< @brief True if the data link is loss */
+        uint32_t mavlink_count{false}; /**< @brief Number of Mavlink connections providing setpoints (implying joystick connected). Should be < 2.*/
```

Comment:
We should initialize the `mavlink_count` with the correct initialization value, i.e. 0, rather than `false`.

Reply:
Whoopsies